sdk/samples/router/general.py

```diff
@@ -0,0 +1,265 @@
+                )
+            ],
+            routes=[
+                'woof'
+            ]
+        ),
+        TrafficRule(
+            conditions=[
+                HeaderTrafficRuleCondition(
+                    field='sheep',
+                    values=['name']
+                )
+            ],
+            routes=[
+                'baaa'
+            ]
+        ),
+        TrafficRule(
+            conditions=[
+                HeaderTrafficRuleCondition(
+                    field='pig',
+                    values=['name']
+                )
+            ],
+            routes=[
+                'oink'
+            ]
+        ),
+        TrafficRule(
+            conditions=[
+                PayloadTrafficRuleCondition(
+                    field='sus',
+                    values=['body']
+                )
+            ],
+            routes=[
+                'meow',
+                'woof',
+                'baaa',
+                'oink',
+                'ring-ding-ding'
+            ]
+        )
+    ]
+
+    # # Create an experiment config (
+    experiment_config = ExperimentConfig(
+        type="xp",
+        config={
+            'variables':
+                [
+                    {'name': 'farm_id', 'field': 'farm_id', 'field_source': 'header'},
+                    {'name': 'country_code', 'field': 'country', 'field_source': 'header'},
+                    {'name': 'latitude', 'field': 'farm_lat', 'field_source': 'header'},
+                    {'name': 'longitude', 'field': 'farm_long', 'field_source': 'header'}
+                ],
+            'project_id': 102
+        }
+    )
+
+    # # Create a resource request config for the router
+    resource_request = ResourceRequest(
+        min_replica=0,
+        max_replica=2,
+        cpu_request="500m",
+        memory_request="512Mi"
+    )
+
+    # # Create a log config for the router
+    log_config = LogConfig(
+        result_logger_type=ResultLoggerType.NOP
+    )
+
+    # # Create an enricher for the router
+    enricher = Enricher(
+        image="asia.gcr.io/gods-dev/echo:1.0.2",
+        resource_request=ResourceRequest(
+            min_replica=0,
+            max_replica=2,
+            cpu_request="500m",
+            memory_request="512Mi"
+        ),
+        endpoint="/",
+        timeout="60ms",
+        port=8080,
+        env=[
+            EnvVar(
+                name="humans",
+                value="farmer-joe"
+            )
+        ]
+    )
+
+    # # Create an ensembler for the router
+    ensembler = DockerRouterEnsemblerConfig(
+        id=1,
+        image="asia.gcr.io/gods-dev/echo:1.0.2",
+        resource_request=ResourceRequest(
+            min_replica=1,
+            max_replica=3,
+            cpu_request="500m",
+            memory_request="512Mi"
+        ),
+        endpoint="/echo",
+        timeout="60ms",
+        port=8080,
+        env=[],
+    )
+
+    # # Create the RouterConfig instance
```

Comment:
nit: bunch of extra `#` for the comments

Reply:
Ok I've fixed the `#`es, thanks for pointing that out!